### ISSUE TITLE
Adding other Archive Access schemas

### ISSUE THREAD

**Issue by HughP** [digitallinguistics/spec#429]
la meta is an application that create MIDI files. Which contain metadata for ingest into an archive. It seems that rather than encoding access paradigms directly into the schema, that it might be more flexible if a framework for adding access schemas was all that was there. There are other schemas which apply to other archives. Alternatively the schema could be updated whenever a new archive is added. 

[1]: https://sites.google.com/site/metadatatooldiscussion/home

**Comment by dwhieb** [comment]
I agree. I didn't love the idea of specifying access schemas into the specification. Do you have ideas for the kind of framework you suggest? How would this work?

**Comment by HughP** [comment]
Ask @hatton what he thinks for describing archive access schemas. He works on Lameta a tool for language archives. As for how to link the two objects, one design pattern I am aware of is the alignment object in LRMI. https://dublincore.org/specifications/lrmi/1.1/

**Comment by hatton** [comment]
This is what lameta has: https://github.com/onset/lameta/blob/master/app/model/Project/AuthorityLists/AccessProtocols/AccessProtocols.json

**Comment by dwhieb** [comment]
@HughP @hatton This is great - thanks! If Lameta is already maintaining an official list of access protocols like this, and plans to maintain it as a public standard, it'd be great to point the DaFoDiL spec to that instead of maintaining our own version of it. Or likewise DaFoDiL could commit to maintaining that list as an official standard, so that Lameta could point to it.

I like the inclusion of a `documentation` property, by the way.
